README.md

```diff
@@ -1,24 +1,58 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails Blog
 
-Things you may want to cover:
+> A simple blog application based on ruby on rails.
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby
+- Rails
+- HTML&CSS
 
-* Database creation
+## Getting Started
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+### Prerequisites
+- Ruby
+- Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+### Setup
+- `git clone` this repo
+- `cd` into the generated directory
+- Run `bundle install`
+- Run `rails db:create` 
+- Run `rails db:migrate` 
+- Run `rails db:seed` 
+- Run `rails s` 
+Note: In case this doesn't work, try `rails db:drop` and do the steps all over again
 
-* Deployment instructions
+### Install
+- Run `bundle install` to install all the necessary dependencies
 
-* ...
+### Usage
+- Run `rails s` to expose a local webserver
+- 
+## Authors
```

Comment:
[Optional] Why don't we use Author instead of Authors since it's a solo project? 🤔